src/web/lib/getZIndex.tsx

```diff
@@ -27,7 +27,7 @@ const indices = [
 	'banner',
 	'dropdown',
 	'burger',
-	'stickyNav',
+	'expanded-veggie-menu',
```

Comment:
`expanded-veggie-menu` makes more sense as it is used in both, stick and classic Nav